### ISSUE TITLE
allow step/title names longer than 256 characters for cucumber

### ISSUE THREAD

**Issue by npopov-d3** [reportportal/reportportal#618]
**Describe the bug**
Cucumber reports fail to complete, due to events being rejected by RP server:

Some of my features are working fine, but others get rejected by RP server (4.2.2) like so:
(node:95882) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 68): Error: Request failed with status code 400: {"error_code":4001,"message":"Incorrect Request. [Field 'name' should have size from '1' to '256'.] "}
Visually what this does is the step never finishes, and later tests/steps start throwing new errors like
Error: Request failed with status code 406: {"error_code":40022,"message":"Finish test item is not allowed. Test item '5cba100da7b11b0001ae3ee3' has descendants with 'IN_PROGRESS' status.

The features that fail have steps which take json strings as params, so they can easily exceed 256 characters.
"reportportal-agent-cucumber": "^2.0.6", "cucumber": "^4.2.1" RP 4.2.2
 Basically cucumber allows you to pass data from example tables which becomes part of the step name, but RP rejects anything over 256ch long.

**Expected behavior**
Steps with long names complete and dont appear hung

**Versions:**
 - Version of RP [4.2.2]

**Comment by DzmitryHumianiuk** [comment]
**resolution:**
increase to 1024

**Comment by npopov-d3** [comment]
can we get this fixed sooner, not in the next major version please? No idea when 5.0 will be out...

**Comment by evjlobanova** [comment]
Will be available in the beta release of version 5

**Comment by evjlobanova** [comment]
https://github.com/reportportal/service-ui/pull/1607
https://github.com/reportportal/service-api/pull/772

**Comment by i-vasilev** [comment]
Hello! @DzmitryHumianiuk @evjlobanova 
I lack the size of step name. 1024 is very small size for big parametrized steps. Can you implement possibility to set size of step name in settings or properties?

**Comment by HardNorth** [comment]
@i-vasilev no.
See:
https://github.com/reportportal/reportportal/issues/1359
https://github.com/reportportal/client-java/issues/154
